JSDemos/Demos/TreeList/RowEditing/description.md

```diff
@@ -1 +1,16 @@
-This example demonstrates editing in _&laquo;row&raquo;_ mode. Each row is&nbsp;supplied with an _&laquo;Edit&raquo;_ link, a&nbsp;click on&nbsp;which switches the row into the editing state. Editors vary from one cell to&nbsp;another: they may be&nbsp;calendars, select boxes, simple text boxes, etc. Note that you can also add or&nbsp;delete a&nbsp;row using the _&laquo;Add a&nbsp;row&raquo;_ button or&nbsp;the _&laquo;Delete&raquo;_ link, respectively.
\ No newline at end of file
+The TreeList allows users to edit data in multiple modes. This demo shows the *"row"* edit mode. To edit any row, click Edit in it. Only one row can be in the edit state at a time.
+
+To enable row edit mode, set the [mode](/Documentation/ApiReference/UI_Components/dxTreeList/Configuration/editing/#mode) property to *"row"* and enable the [editing](/Documentation/ApiReference/UI_Components/dxTreeList/Configuration/editing/#mode) object's [allowUpdating](/Documentation/ApiReference/UI_Components/dxTreeList/Configuration/editing/#allowUpdating), [allowAdding](/Documentation/ApiReference/UI_Components/dxTreeList/Configuration/editing/#allowAdding), and [allowDeleting](/Documentation/ApiReference/UI_Components/dxTreeList/Configuration/editing/#allowDeleting) properties to allow edit operations.
+
+Buttons for allowed edit operations are displayed in an edit column. To customize it programmatically, declare a column of the `buttons` [type](/Documentation/ApiReference/UI_Components/dxTreeList/Configuration/columns/#type) and specify available buttons. In this demo, the `buttons` column is used to hide an Add button from each row. See the following help topic for more details on edit column customization: [Customize the Edit Column](/Documentation/Guide/UI_Components/TreeList/Columns/Column_Types/Command_Columns/#Customize_the_Edit_Column).
```

Comment:
The sentence
> To customize it programmatically, declare a column of the `buttons` [type](/Documentation/ApiReference/UI_Components/dxTreeList/Configuration/columns/#type) and specify available buttons.

Refers to an edit column mentnioned in the previous sentence, correct? If so, should I create another column of the `buttons` type or redefine the mentioned edit column?

Reply:
This sentence refers to the edit column mentioned in the previous sentence, but this column is not in the columns array because declaring editing.allow... properties creates the edit column _implicitly_. To customize this column, a programmer should declare a column of the `buttons` type add add desired properties. Thus there can be only one column of the buttons type.

Reply:
Alright, let's keep the current version.

Reply:
```suggestion
Buttons for allowed edit operations are displayed in an edit column. To customize it programmatically, declare a column of the `buttons` [type](/Documentation/ApiReference/UI_Components/dxTreeList/Configuration/columns/#type) and specify the available buttons. In this demo, the `buttons` column is used to hide the Add button in each row. See the following help topic for more details on edit column customization: [Customize the Edit Column](/Documentation/Guide/UI_Components/TreeList/Columns/Column_Types/Command_Columns/#Customize_the_Edit_Column).
```